### PR TITLE
4 components rendering

### PR DESCRIPTION
**Pull request by Siege211**

### CODE REVIEW

**Comment by janeslee** [comment]
LGTM TEAM!

**Comment by stevens-works** [comment]
this is amazing! but you forgot a semi-colon on line 3

**Comment by shakabandai** [comment]
YES! YES! YES!

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	},
+	module: {
+		rules: [
+			{
+        test: /\.jsx?/,
+        include: SRC_DIR,
+        loader: 'babel-loader',
+        query: {
+          presets: ['react']
+        }
```
Comment (therobinkim) [inline webpack.config.js:19]: don't mix and match tabs & spaces
stick with just one

**Comment by therobinkim** [comment]
What is the purpose of this `Master-Proxy` repo?

**Comment by therobinkim** [comment]
You're committing random files like `.DS_Store` and your webpack output
Either stop using `git add .` (or similar) or fix your `.gitignore` to not add these files

File: Components/Jane-MusicPlayer/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Jane_MusicPlayer_Component
```
Comment (therobinkim) [inline Components/Jane-MusicPlayer/README.md:1]: I don't think any individual's component files should be in the proxy repo 🤔

**Pull request merged**